app/models/doc_result.rb

```diff
@@ -15,6 +15,20 @@ def self.fromSearch(idval, typeval, urlval, titleval, sumval, scoreval)
     obj
   end
 
+  def self.fromLocationSearch(idval, typeval, urlval, titleval, sumval, scoreval,latitude,longitude,blurred)
```

Comment:
Hi, just wondering if we could tidy up the spacing towards the end of this line, and on lines 41-2 below. This really helps keep the code tidy over the long term, thanks!

Reply:
correcting this..